php-http/httplug-bundle/1.6/etc/packages/httplug.yaml

```diff
@@ -0,0 +1,16 @@
+httplug:
+    plugins:
+        logger:
+            enabled: true
```

Comment:
or maybe use %kernel.debug% as default and add a comment that you can set this to true if you want to log HTTP requests in production?